src/main/java/me/moodcat/api/models/SongModel.java

```diff
@@ -73,6 +73,24 @@
     private double arousal;
 
     /**
+     * The link to the site to purchase the song.
+     *
+     * @param purchaseUrl
+     *          The new purchaseURL to set.
```

Comment:
I would delete the new in the javadoc (wich is almost all javadocs)
This insist that this isn't used for a initial purchaseUrl.